src/checkout/checkout-store-error-selector.ts

```diff
@@ -322,71 +193,125 @@ export default class CheckoutStoreErrorSelector {
+                }
+            }
+        }
+    );
+
+    const getSelectShippingOptionError = createSelector(
+        ({ shippingStrategies }: InternalCheckoutSelectors) => shippingStrategies.getSelectOptionError,
+        ({ consignments }: InternalCheckoutSelectors) => consignments.getUpdateShippingOptionError,
+        (getSelectOptionError, getUpdateShippingOptionError) => (consignmentId?: string) => {
+            return (
+                getSelectOptionError() ||
+                getUpdateShippingOptionError(consignmentId)
+            );
+        }
+    );
+
+    return memoizeOne((
+        state: InternalCheckoutSelectors
+    ): CheckoutStoreErrorSelector => {
+        const selector = {
+            getLoadCheckoutError: state.checkout.getLoadError,
+            getUpdateCheckoutError: state.checkout.getUpdateError,
+            getSubmitOrderError: state.paymentStrategies.getExecuteError,
+            getFinalizeOrderError: state.paymentStrategies.getFinalizeError,
+            getLoadOrderError: state.order.getLoadError,
+            getLoadCartError: state.cart.getLoadError,
+            getLoadBillingCountriesError: state.countries.getLoadError,
+            getLoadShippingCountriesError: state.shippingCountries.getLoadError,
+            getLoadPaymentMethodsError: state.paymentMethods.getLoadError,
+            getLoadPaymentMethodError: state.paymentMethods.getLoadMethodError,
+            getInitializePaymentError: state.paymentStrategies.getInitializeError,
+            getSignInError: state.customerStrategies.getSignInError,
+            getSignOutError: state.customerStrategies.getSignOutError,
+            getInitializeCustomerError: state.customerStrategies.getInitializeError,
+            getLoadShippingOptionsError: state.consignments.getLoadShippingOptionsError,
+            getSelectShippingOptionError: getSelectShippingOptionError(state),
+            getContinueAsGuestError: state.billingAddress.getContinueAsGuestError,
+            getUpdateBillingAddressError: state.billingAddress.getUpdateError,
+            getUpdateShippingAddressError: state.shippingStrategies.getUpdateAddressError,
+            getDeleteConsignmentError: state.consignments.getDeleteError,
+            getUpdateConsignmentError: state.consignments.getUpdateError,
+            getCreateConsignmentsError: state.consignments.getCreateError,
+            getInitializeShippingError: state.shippingStrategies.getInitializeError,
+            getApplyCouponError: state.coupons.getApplyError,
+            getRemoveCouponError: state.coupons.getRemoveError,
+            getApplyGiftCertificateError: state.giftCertificates.getApplyError,
+            getRemoveGiftCertificateError: state.giftCertificates.getRemoveError,
+            getLoadInstrumentsError: state.instruments.getLoadError,
+            getDeleteInstrumentError: state.instruments.getDeleteError,
+            getLoadConfigError: state.config.getLoadError,
+        };
+
+        return {
```

Comment:
How come these 2 are not combined in a single return?

Reply:
This is because I have to pass `selector` into `getError`